src/main/java/net/minecraftforge/registries/GameData.java

```diff
@@ -302,6 +308,12 @@ public void onAdd(IForgeRegistryInternal<Block> owner, RegistryManager stage, in
             @SuppressWarnings("unchecked")
             ClearableObjectIntIdentityMap<IBlockState> blockstateMap = owner.getSlaveMap(BLOCKSTATE_TO_ID, ClearableObjectIntIdentityMap.class);
 
+            if (oldBlock != null)
+            {
+                for (IBlockState state : oldBlock.getBlockState().getValidStates())
```

Comment:
I'd really prefer braces on `for` loops, most of them in the forge codebase have braces